Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/qrcode/AutoCheckoutHelper.kt

```diff
@@ -0,0 +1,68 @@
+package de.rki.coronawarnapp.eventregistration.checkins.qrcode
```

Comment:
could we not call it helper? maybe DefaultAutoCheckoutLength?

Reply:
I can change it, but what's wrong with "helper"?